client/my-sites/plans-features-main/index.jsx

```diff
@@ -315,29 +335,67 @@ class PlansFeaturesMain extends Component {
 		);
 	}
 
+	getIntervalTypeToggle() {
+		const {
+			translate,
+			intervalType,
+			site,
+			basePlansPath,
+		} = this.props;
+		const segmentClasses = classNames(
+			'plan-features__interval-type',
+			'price-toggle'
+		);
+
+		let plansUrl = '/plans';
+		if ( basePlansPath ) {
+			plansUrl = basePlansPath;
+		}
+
+		return (
+			<SegmentedControl compact className={ segmentClasses } primary={ true }>
+				<SegmentedControlItem
+					selected={ intervalType === 'monthly' }
+					path={ plansLink( plansUrl, site, 'monthly' ) }
+				>
+					{ translate( 'Monthly billing' ) }
+				</SegmentedControlItem>
+
+				<SegmentedControlItem
+					selected={ intervalType === 'yearly' }
+					path={ plansLink( plansUrl, site, 'yearly' ) }
+				>
+					{ translate( 'Yearly billing' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Billing' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).